packages/synapse-react-client/src/lib/containers/discussion_forum/DiscussionReply.tsx

```diff
@@ -88,24 +88,20 @@ export const DiscussionReply: React.FC<DiscussionReplyProps> = ({
           )}
         </>
       )}
-      <Modal
-        size="lg"
-        show={showReplyModal}
-        onHide={() => setShowReplyModal(false)}
-        animation={false}
-      >
-        <Modal.Header>
-          <Modal.Title>Edit Reply</Modal.Title>
-        </Modal.Header>
-        <Modal.Body>
+      <DialogBase
+        maxWidth="md"
+        open={showReplyModal}
+        onCancel={() => setShowReplyModal(false)}
+        title="Edit Reply"
```

Comment:
main | feature
:---:|:---:
![main_DiscussionReply_EditReply](https://user-images.githubusercontent.com/26949006/236285210-0cee3c55-7097-44d9-b1e8-14e367c23c6f.png) | ![feature_DiscussionReply_EditReply](https://user-images.githubusercontent.com/26949006/236285188-91cdf414-11d5-46c9-882c-0c9da8cb0654.png)

Reply:
This dialog appears to have extra top and bottom padding, and the bottom horizontal line seems out of place (since the buttons are not down there).  Maybe hide if no content?

Reply:
Good call! I originally passed the component that includes the buttons (ForumThreadEditor) as the content of a dialog, but that puts the buttons above the divider. Since the component appears in several places as a dialog, I refactored the component so it handles rendering the dialog, rather than passing the entire component as the content of the dialog. Now the style matches other confirmation dialogs, where the Cancel/Save buttons appear below the divider line. Happy to change things back though and hide the divider if there's no action buttons, if that's preferred!

![feature_DiscussionReply_EditReply](https://user-images.githubusercontent.com/26949006/236353799-264cc8aa-f60c-47ba-8e88-6dcad8ede235.png)